mollie/common_types.go

```diff
@@ -115,32 +115,147 @@ type PaginationLinks struct {
 	Documentation *URL `json:"documentation,omitempty"`
 }
 
-// CategoryCode specifies an industry or category.
-type CategoryCode uint
+// BusinessCategory specifies an industry or category.
+type BusinessCategory string
 
-// Available category codes.
+// Available business categoreis.
 const (
-	BookMagazinesAndNewspapers          CategoryCode = 5192
```

Comment:
Similar to the 'CategoryCode' type, I want to maintain these while changing the block comment to say that they are "historic" and shouldn't be used any longer.